lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -210,23 +210,26 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
 
     // Filmstrip
     const timelineEl = this.dom.createChildOf(element, 'div', 'lh-filmstrip-container');
-    const thumbnailAudit = category.auditRefs.find(audit => audit.id === 'screenshot-thumbnails');
-    const thumbnailResult = thumbnailAudit && thumbnailAudit.result;
-    if (thumbnailResult && thumbnailResult.details) {
-      timelineEl.id = thumbnailResult.id;
-      const filmstripEl = this.detailsRenderer.render(thumbnailResult.details);
-      filmstripEl && timelineEl.appendChild(filmstripEl);
+    // We only expect one of these, but the renderer will support multiple
+    const thumbnailAudits = category.auditRefs.filter(audit => audit.group === 'filmstrip');
```

Comment:
Is this breaking? v7?

Reply:
yeah this would break formatting the v6 reports in a new renderer. good call. i'll add a compat bit to also the `screenshot-thumbnails` audit